src/solver/conv_direct_naive_conv.cpp

```diff
@@ -129,12 +129,12 @@ std::string ConvDirectNaiveConvKernelName(const ProblemDescription& problem)
 #ifndef NDEBUG // enable in debug mode only
     if(env::enabled(MIOPEN_DEBUG_CONV_DIRECT_NAIVE_USE_PACKED_KERNELS))
     {
-        kernel_name << "naive_conv_packed_";
+        kernel_name << "naive_conv_ab_packed_";
```

Comment:
[Notice]  Just in case: "packed" kernels are not renamed in this PR (yet). Of course, I see that this is draft..

Reply:
@atamazov renamed both kernel names